wm_rhasspy_ros_bridge/listen listen
---
bool sucess